package/base-files/files/sbin/sysupgrade

```diff
@@ -118,32 +118,41 @@ list_conffiles() {
 		BEGIN { conffiles = 0 }
 		/^Conffiles:/ { conffiles = 1; next }
 		!/^ / { conffiles = 0; next }
-		conffiles == 1 { print }
+		# in format: <space><filename><space><hash>
+		conffiles == 1 { print $(NF) "  " substr($0,2,length($0)-length($(NF))-2) }
```

Comment:
list_conffiles here will return the same format sha256sum returns (and expects while checking a hash)